themes/edx.org/lms/templates/footer.html

```diff
@@ -35,10 +37,10 @@ <h2 class="sr footer-about-title">${_("About edX")}</h2>
               % endfor
           </nav>
           <p class="copyright">${_(
-          u"\u00A9 2016 edX Inc.  All rights reserved except where noted.  "
+          u"\u00A9 {year} edX Inc.  All rights reserved except where noted.  "
```

Comment:
@schenedx not in this case. I explained to @MatthewPiatetsky offline that `now` is a Django template tag, not part of Mako (so we can't use it).